IDE/Espressif/ESP-IDF/examples/template/components/wolfssl/include/user_settings.h

```diff
@@ -1058,8 +1058,8 @@ Turn on timer debugging (used when CPU cycles not available)
             #error "USE_CERT_BUFFERS_1024 is already defined. Pick one."
         #endif
 
-        /* Be sure to include in app when using example certs: */
-        #include <wolfssl/certs_test.h>
+        /* Be sure to include in app, not here, when using example certs: */
+        /* #include <wolfssl/certs_test.h> */
```

Comment:
Is this PR really needed anymore? You set `USE_CERT_BUFFERS_256` and even try and use `ca_cert_der_2048` below. Shouldn't certs_test.h be needed? Maybe the USE_CERT_BUFFERS_ should not be part of the user_settings.h?

Reply:
Done